static/app/views/issueList/searchBar.tsx

```diff
@@ -93,6 +98,12 @@ const EXCLUDED_TAGS = ['environment'];
 function IssueListSearchBar({organization, tags, ...props}: Props) {
   const api = useApi();
   const {selection: pageFilters} = usePageFilters();
+  const {tags: issueTags} = useFetchIssueTags({
```

Comment:
Can you pass in the time range filters here? It should work similarly to `tagValueLoader` below. You could also do this in a followup PR if you'd like